samza-core/src/main/scala/org/apache/samza/util/CoordinatorStreamUtil.scala

```diff
@@ -110,13 +111,34 @@ object CoordinatorStreamUtil extends Logging {
       jobConfig.getJobId)
   }
 
+  /**
+   * Reads and returns launch config persisted in coordinator stream. Only job.auto sizing configs are currently supported.
+   * @param config full job config
+   * @param metadataStore an instance of the instantiated MetadataStore
+   * @return empty config if auto sizing is disabled, otherwise auto sizing related configs.
+   */
+  def readLaunchConfigFromCoordinatorStream(config: Config, metadataStore: MetadataStore): Config = {
```

Comment:
should we call it readAutosizingConfigs?

Reply:
We can make it specific to autosizing too, or we can make it generic so we may expand this method to include more when necessary, which do you prefer?